juju/juju
# [JUJU-3494] Upgrade cmd to v3.0.9

This PR updates the cmd package to v3.0.9. This version enables the `documentation` command to use a list of ids for every command. This is required to finish the works around the publication of the markdown rendered commands in discourse (see #15406). Additionally, I have cleaned some misleading references to commands no longer available.

## Checklist


## QA steps

Render the markdown documentation in a local folder and check that the ids are correctly replaced.

```sh
mkdir /tmp/docs
echo "get the ids file"
 wget https://raw.githubusercontent.com/juju/juju/fd40f7aa723883a6da41dd7a804f1e52724f4fc4/scripts/discourse-sync/post_ids.yaml -O /tmp/ids.txt
_build/linux_amd64/bin/juju documentation --split --no-index --discourse-ids=/tmp/ids.txt --out /tmp/docs
```
Now check the ids are correctly replaced.
```sh
grep update-cloud /tmp/ids.txt
grep update-cloud /tmp/docs/add-cloud.md
```
